server/src/main/java/school/journal/repository/impl/SubjectInScheduleRepository.java

```diff
@@ -0,0 +1,44 @@
+package school.journal.repository.impl;
+
+import org.hibernate.Criteria;
+import org.hibernate.Session;
+import org.hibernate.criterion.Criterion;
+import school.journal.entity.SubjectInSchedule;
+import school.journal.repository.IRepository;
+import school.journal.repository.exception.RepositoryException;
+import school.journal.repository.specification.HibernateSpecification;
+
+import java.util.List;
+
+/**
+ * Created by zasam on 20.02.2017.
+ */
```

Comment:
delete such comments, edit your template for java files in IDEA, make it without this comment